solrsearch-plugins/docs/SolrSearch-batchsink.md

```diff
@@ -0,0 +1,75 @@
+# SolrSearch Batch Sink
+
+
+Description
+-----------
+The Solr search batch sink takes the structured record from the input source and indexes it into the Solr server or
+SolrCloud using the collection and key field specified by the user.
+
+The incoming fields from the previous stage in pipelines are mapped to Solr fields. Also, user is able to specify the
+mode of the Solr to connect to. For example, Single Node Solr or SolrCloud.
+
+Use Case
+--------
+Solr search batch sink is used to write data to the Solr server or SolrCloud. For example, the plugin can be used in
+conjuction with a stream batch source to parse a file and read its contents in Solr.
+
+Properties
+----------
+**referenceName:** This will be used to uniquely identify this sink for lineage, annotating metadata, etc.
+
+**solrMode:** Solr mode to connect to. For example, Single Node Solr or SolrCloud.
```

Comment:
use exact name "SingleNode" as that's how it looks in the config. doesn't need a space.